src/styles/colors.css

```diff
@@ -98,7 +98,7 @@
     --dark-theme-version-badge-purple: var(--dark-five);
     --dark-theme-icon-color: var(--warm-gray);
     --dark-theme-star-icon-active-color: var(--white-two);
-    --dark-theme-divider: var(--pure-white);
+    --dark-theme-divider: var(--white-two);
```

Comment:
I know it's out of scope of this PR but just from reading I have no idea what `white-two` refers to...

Reply:
Yeah that's reasonable. It just means, the white that comes after the first white.

I think that at various point devs (including me) have run out of appetite for naming the many shades of gray, white, "dark" and purple that have been used in the simularium color scheme...